bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx67.0.5.html

```diff
@@ -70,7 +74,7 @@ <h2>{{ _('Including Firefox Monitor, your watchful eye for data breaches') }}</h
         <div class="benefit-content">
           <h3>{{ _('Products that respect') }}</h3>
           <p>
-            {% trans send='https://send.firefox.com/', lockwise='https://lockbox.firefox.com/', monitor='https://monitor.firefox.com/' %}
+            {% trans send='https://send.firefox.com/', lockwise='https://lockwise.firefox.com/', monitor='https://monitor.firefox.com/' %}
             Firefox products like <a href="{{ send }}">Send</a>, <a href="{{ lockwise }}">Lockwise</a> and <a href="{{ monitor }}">Monitor</a> help you get things done without invading your privacy.
```

Comment:
This string doesn't match the original, which didn't include links, so it's not being translated. I think the only way to fix this is to do it "for real" by updating the lang files. Or else we can remove the links and go back to the original string, but I know marketing will want those links. We should talk to @peiying2 and give her a heads-up.

https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/whatsnew_67.0.5.lang#L83

Reply:
OK, With @peiying2's blessing I think we're ready to move ahead with updating the lang files, so this string is good.